symbols.xml

```diff
@@ -2226,7 +2225,7 @@
         <global-address name='enabler'/>
         <global-address name='flows'/>
         <global-address name='gps'/>
-        <global-address name='gview'/>
+        <global-address name='gview' value='0x1421040'/>
```

Comment:
Oh, all of the globals in the "generated cglobals" section have empty entries in the ".bss compound" section above (at least on *nix). That's because one script copies the example, which has those in the upper section, and another script scans and adds them in the lower section but doesn't get rid of the empty entries. Quietust was looking into that, I think.